### ISSUE TITLE
Redirect search.uiowa.edu to uiowa.edu/search

### ISSUE THREAD

**Issue by briand44** [uiowa/uiowa#7463]
# Story

As an owner of search.uiowa.edu, we would like it to redirect to uiowa.edu/search so that we can retire the search.uiowa.edu application on the LAMP service.

# Proposed solution

- Request the redirect be created on Vulture
- Request hostmaster lower TTL for search.uiowa.edu
- Test Vulture redirect locally once created
- Request hostmaster update DNS for search.uiowa.edu to point to Vulture.its.uiowa.edu
- Probably should keep TTL low for a week or two to give folks time to yell at us.

# Follow up

- Let Hostmaster know they can set TTL back to default for search.uiowa.edu
- Let LAMP folks know search.uiowa.edu application is no longer needed?
- Any necessary cleanup
    - Remove credentials for LAMP service from password manager
    - Archive git repo
    - anything else?

#7464

**Comment by briand44** [comment]
Sean mentioned it would be good to understand how long we plan to keep the redirect. ITS-EI-SAS prefers to keep Vulture redirects no longer than 1 year but have made exceptions.

**Comment by joewhitsitt** [comment]
Did we make a decision on how long to initially request this redirect to stay in place?

**Comment by briand44** [comment]
I don't think we did. I'd be fine with 1 year.

**Comment by joewhitsitt** [comment]
Opened ticket 2101601 with ITS-EI-SAS requesting a vulture redirect for 1 year.

**Comment by joewhitsitt** [comment]
Sent email from its-web to hostmaster requesting TTL be lowered for search.uiowa.edu

Update: TTL was lowered

**Comment by joewhitsitt** [comment]
still waiting on the vulture redirect

**Comment by joewhitsitt** [comment]
Just got notified that it was in place. Tested the redirect locally with /etc/hosts.

**Comment by joewhitsitt** [comment]
Requested hostmaster point the domain to vulture and keep the TTL lowered.

UPDATE: re-pointed and redirected

**Comment by joewhitsitt** [comment]
Looks like SSL is not in place for the redirect. I thought it was a /etc/hosts things locally. I will respond to the ticket with EI-SAS

**Comment by joewhitsitt** [comment]
Going to close this issue as the follow-up has been stubbed out. https://github.com/uiowa/uiowa/issues/7464
